core/src/main/scala/org/scalawag/jibe/mandate/command/package.scala

```diff
@@ -34,4 +36,9 @@ package object command {
   implicit val FileToValue = new ToValue[File, StringValue] {
     override def toValue(a: File): Option[StringValue] = Some(StringValue(a.getPath))
   }
+
+  implicit val DurationToValue = new ToValue[Duration, LongValue] {
+    override def toValue(d: Duration) = Some(new LongValue(d.toSeconds))
```

Comment:
I think it would make more sense to make this milliseconds (since it has that precision) and then compensate in your script by dividing the argument by 1000 in there.